src/utilities/animation.ts

```diff
@@ -0,0 +1,85 @@
+// Adapted from https://gist.github.com/gre/1650294
+export const easeInOutQuad: TimingFunction = (t: number) =>
+    t < .5 ? 2 * t * t : -1 + (4 - 2 * t) * t;
```

Comment:
This seems a little complicated; would we be better off just wiggling as the `sin(time)` intermittently and skipping most of the keyframing stuff? Then again, maybe that's not as configurable...

Reply:
Could css & part be reintroduced down the line when the bug is gone? I guess that'd be hard to do though cause not everyone would update their browsers.